services/vehicle-assembly-process-defect-detection-model-service/app/core/dependencies.py

```
@@ -0,0 +1,375 @@
+import logging
+from typing import Optional
+from fastapi import Depends, HTTPException, status, Request
+from datetime import datetime
+
+from app.core.config import settings
+
+# 로거 설정
+logger = logging.getLogger(__name__)
+
+# 전역 모델 매니저 인스턴스
+_model_manager: Optional['DefectDetectionModelManager'] = None
```

Ruff (0.12.2)

Undefined name `DefectDetectionModelManager`

(F821)